stdlib/public/core/UnsafeRawPointer.swift.gyb

```diff
@@ -835,6 +838,10 @@ public struct Unsafe${Mutable}RawPointer : Strideable, Hashable, _Pointer {
     }
   }
 
+  @available(*, unavailable, renamed: "copy(from:bytes:)")
+  public func copyBytes(from source: UnsafeRawPointer, count: Int)
+  { fatalError("unreachable") }
```

Comment:
i thought dummy blocks weren’t to be formatted like “real” function bodies, like `a.map{ $0.foo }` instead of 

```swift
a.map 
{
    $0.foo 
}
```